packages/twenty-front/folderStructure.json

```diff
@@ -21,7 +21,7 @@
 
     "moduleFolderRule": {
       "name": "{kebab-case}",
-      "folderRecursionLimit": 6,
+      "folderRecursionLimit": 4,
```

Comment:
unrelated to this PR but we need to re-work this lint rule, the impact on performances is too high